.github/workflows/ccpp.yaml

```
@@ -2,9 +2,9 @@ name: C/C++ CI
 
 on:
   push:
-    branches: [ '1.10' ]
+    branches: [ '1.10', '1.11']
```

```suggestion
    branches: [ '1.10', '1.11' ]
```

細かいですが、スペースが消えてしまっていて、ちょっと他と書式が変っちゃってるので念のためコメントしておきます

---

あ、これはupstream側のコード変更なんですね、こちらで不用意に触らない方がいいと思ったので、このsuggestionは取り下げます